networkit/cpp/io/EdgeListReader.cpp

```diff
@@ -20,7 +20,12 @@ namespace NetworKit {
 EdgeListReader::EdgeListReader(char separator, node firstNode, const std::string &commentPrefix,
                                bool continuous, bool directed)
     : separator(separator), commentPrefix(commentPrefix), firstNode(firstNode),
-      continuous(continuous), mapNodeIds(), directed(directed) {}
+      continuous(continuous), mapNodeIds(), directed(directed) {
+    if (!continuous && firstNode != 0) {
+        WARN("firstNode set to 0 since continuous is true");
```

Comment:
This should be `false`, not `true`.
```suggestion
        WARN("firstNode set to 0 since continuous is false");
```